runtime/service.go

```diff
@@ -928,23 +929,29 @@ func (s *service) Delete(requestCtx context.Context, req *taskAPI.DeleteRequest)
 	if req.ExecID != "" {
 		return resp, nil
 	}
+	var result *multierror.Error
+	// Trying to release stub drive for further reuse
+	err = s.containerStubHandler.Release(requestCtx, req.ID, s.driveMountClient, s.machine)
+	if err != nil {
+		return nil, multierror.Append(result, errors.Wrapf(err, "failed to release stub drive for container: %s", req.ID))
```

Comment:
The way I think we want to use `multierror` here is to just store the error returned by `Release` rather than return it right away. So something more like this:
```
err = s.containerStubHandler.Release(...)
if err != nil {
    result = multierror.Append(result, errors.Wrapf(err, ...))
}
```

The rest of the code can stay as returns though as the previous logic was fine.